build/packages/rpm/server/SPECS/pmm-update.spec

```diff
@@ -63,6 +63,9 @@ install -p -m 0755 bin/pmm-update %{buildroot}%{_sbindir}/
 
 
 %changelog
+* Thu Dec 8 2022 Michal Kralik <michal.kralik@percona.com> - 2.29.0-2
+- Migrated pmm-update to monorepo
```

Comment:
```suggestion
- PMM-11207 Migrate pmm-update to monorepo
```

Reply:
I suggest we don't include PMM-11207 in the changelog. The ticket number does not say anything to anyone outside Percona.

Reply:
Why? Our Jira is public, so far these references worked well... With a reference like this it's easier to navigate through the history and, even more, oftentimes it facilitates the troubleshooting, which can save time.